packages/trpc/server/routers/viewer/teams/inviteMember/utils.ts

```diff
@@ -331,52 +380,135 @@ export function getIsOrgVerified(
   } as { isInOrgScope: false; orgVerified: never; autoAcceptEmailDomain: never };
 }
 
-export async function createAndAutoJoinIfInOrg({
+export function shouldAutoJoinIfInOrg({
```

Comment:
Import in inviteMemberUtils.test.ts needs to be fixed: 
![Screenshot 2023-11-22 at 12 59 28 PM](https://github.com/calcom/cal.com/assets/30310907/13cf0ada-bc85-49fc-98aa-a3836cd5238f)